### PR TITLE
Release/2.0.2

### PR DESCRIPTION
**Pull request by tinhvqbk**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -103,7 +104,7 @@ Full customizable variables:
   --colorFocusInformation: #375de733;
   --colorInformationDark: #2c4ab8;
   --colorInformationDarker: #21388b;
-  --fontFamily: Roboto,'Helvetica Neue',Helvetica,Arial,sans-serif;
+  --fontFamily: roboto, 'Helvetica Neue', helvetica, arial, sans-serif;
```
Comment (namdaoduy) [inline README.md]: Why we change this? @tinhvqbk

File: scss/utilities/_deprecated.scss [hunk 2]
```diff
@@ -0,0 +1,135 @@
+/*!
+ * Deprecated Utilities class
+ * Since 2.0.2
```
Comment (namdaoduy) [inline scss/utilities/_deprecated.scss:3]: @tinhvqbk You can put messages inside your css rulesets like so:

```
.center {
  text-align: "center";
  --deprecated: "WARNING: Use .other-class instead";
}
```
This way developers and users can see the message directly in their browsers and take action

**Review by namdaoduy — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (tinhvqbk) [reply README.md]: It seems like issues from stylelint ignore

File: scss/utilities/_deprecated.scss (hunk 2 above)
Comment (tinhvqbk) [reply scss/utilities/_deprecated.scss:3]: Ok

File: README.md (hunk 1 above)
Comment (tinhvqbk) [reply README.md]: fixed on https://github.com/gotitinc/aha-css/commit/b5983bf953b651275b2a73511c6d4b231d8889d5 @namdaoduy

File: scss/utilities/_deprecated.scss (hunk 2 above)
Comment (tinhvqbk) [reply scss/utilities/_deprecated.scss:3]: added on https://github.com/gotitinc/aha-css/commit/04423e3429e8aeb83bf434e04838a40573f38f7f @namdaoduy

**Review by tinhvqbk — COMMENTED** [review]
fixed

File: README.md [hunk 3]
```diff
@@ -103,7 +104,7 @@ Full customizable variables:
   --colorFocusInformation: #375de733;
   --colorInformationDark: #2c4ab8;
   --colorInformationDarker: #21388b;
-  --fontFamily: Roboto,'Helvetica Neue',Helvetica,Arial,sans-serif;
+  --fontFamily: Roboto, 'Helvetica Neue', helvetica, arial, sans-serif;
```
Comment (namdaoduy) [inline README.md]: Is this an autofix? @tinhvqbk

**Review by namdaoduy — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (tinhvqbk) [reply README.md]: It's manual, but missing some things, let's me fix it

Comment (tinhvqbk) [reply README.md]: fixed https://github.com/gotitinc/aha-css/commit/959a0c8421ac6e39685f6ad198c32597890ac141

**Review by namdaoduy — APPROVED** [review]

**Pull request merged**